### PR TITLE
R tweaks

### PR DESCRIPTION
**Pull request by jeroenooms**
Some tweaks for `language: r`. @jimhester @craigcitro

### CODE REVIEW

**Comment by jimhester** [comment]
The code looks good to me.

It might be worth adding a new test case to ensure no-manual is passed to R CMD build when LaTeX is disabled.

The spec file for R is located at

https://github.com/travis-ci/travis-build/blob/master/spec/build/script/r_spec.rb

**Comment by jeroenooms** [comment]
@jimhester Thanks. I added the unit test. Can you pls review?

File: spec/build/script/r_spec.rb [hunk 1]
```diff
@@ -139,6 +139,12 @@
                          echo: true, timing: true]
   end
 
+  it 'skips PDF manual when LaTeX is disabled' do
+    data[:config][:latex] = false
+    should include_sexp [:cmd, /.*R CMD check.* --no-manual.*/
```
Comment (craigcitro) [inline spec/build/script/r_spec.rb]: I think you need a `,` at the end of the line?

**Comment by craigcitro** [comment]
Also LGTM modulo a small typo fix.

**Comment by jeroenooms** [comment]
Thanks, fixed the typo!

**Comment by craigcitro** [comment]
Awesome -- @BanzaiMan this looks good when you get a chance to merge.

**Pull request merged**